CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # UNRELEASED
 
+**BREAKING CHANGES**
+
+-   Update matrix-rusk-sdk to `0.9.0`.
```

Comment:
@andybalaam Should I use a git hash here too? even it is this actual tag, or link the tag?

Reply:
I think what you have is sufficient.